lib/DirectoryWatcher.js

```diff
@@ -106,13 +106,14 @@ class DirectoryWatcher extends EventEmitter {
 						}
 					}
 				};
+				this.watcher.on("change", this.onWatchEvent.bind(this));
```

Comment:
I am afraid about perf here...

Reply:
Didn't end up being necessary anyway, the new solution is simpler